src/shared/components/palette/utils/date.pipe.ts

```diff
@@ -24,20 +33,35 @@ export class DatePipe implements PipeTransform {
       } else {
         offsetDate = this.getDate(match);
       }
-      // RDM-1149 changed the pipe logic so that it doesn't add an hour to 'Summer Time' dates on DateTime field type
-
-      resultDate = `${offsetDate.getDate()} ${DatePipe.MONTHS[offsetDate.getMonth()]} ${offsetDate.getFullYear()}`;
-      if (match[4] && match[5] && match[6] && format !== 'short') {
-        resultDate += ', ';
-        resultDate += this.getHour(offsetDate.getHours().toString()) + ':';
-        resultDate += this.pad(offsetDate.getMinutes()) + ':';
-        resultDate += this.pad(offsetDate.getSeconds()) + ' ';
-        resultDate += (this.toInt(offsetDate.getHours().toString()) >= 12) ? 'PM' : 'AM' ;
+      // 'short' format is meaningful to formatDate, but not the same meaning as in the unit tests
+      if (this.formatTrans && format && format !== 'short') {
+        // support for java style formatting strings for dates
+        format = this.translateDateFormat(format);
+        // we specify UTC because we've already done timezone offsetting if the zone === local
+        resultDate = formatDate(offsetDate, format, 'en-UK', null)
```

Comment:
I thought the timezone was `en-GB`? Might be inconsequential but thought I'd check